src/JKang.IpcServiceFramework.Client/IpcClientOptions.cs

```diff
@@ -12,7 +12,18 @@ public class IpcClientOptions
         /// The number of milliseconds to wait for the server to respond before
         /// the connection times out. Default value is 60000.
         /// </summary>
-        public int ConnectionTimeout { get; set; } = 60000;
+        public int ConnectionTimeout { get; set; } = 60000;
+
+        /// <summary>
+        /// Indicates the method that will be used during deserialization on the server for locating and loading assemblies.
+        /// If <c>false</c>, the assembly used during deserialization must match exactly the assembly used during serialization.
+        /// 
+        /// If <c>true</c>, the assembly used during deserialization need not match exactly the assembly used during serialization.
+        /// Specifically, the version numbers need not match.
+        /// 
+        /// Default is <c>false</c>.
+        /// </summary>
+        public bool UseSimpleTypeNameAssemblyFormatHandling { get; set; } = false;
```

Comment:
This name is based on the Netwonsoft JSON serializer option for consistency https://www.newtonsoft.com/json/help/html/T_Newtonsoft_Json_TypeNameAssemblyFormatHandling.htm